<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-CF98PN2R" name="scripts._01_ProjectTest" tests="11" failures="1" timestamp="2023-06-12T17:06:39 MDT" time="53.998" errors="1">
  <testcase name="validateAddress" time="5.023" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateMessage" time="5.588" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="setUp" time="5.491" classname="scripts._01_ProjectTest">
    <error type="org.openqa.selenium.SessionNotCreatedException" message="session not created
from no such execution context: uniqueContextId not found
  (Session info: chrome=114.0.5735.110)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;LAPTOP-CF98PN2R&#039;, ip: &#039;192.168.1.70&#039;, os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_361&#039;
Driver info: driver.version: Driver
remote stacktrace: Backtrace:
	GetHandleVerifier [0x006EA813+48355]
	(No symbol) [0x0067C4B1]
	(No symbol) [0x00585358]
	(No symbol) [0x005789B0]
	(No symbol) [0x00577138]
	(No symbol) [0x005777AA]
	(No symbol) [0x00588417]
	(No symbol) [0x005899E1]
	(No symbol) [0x00589A80]
	(No symbol) [0x005D1F1B]
	(No symbol) [0x005D093C]
	(No symbol) [0x005CA536]
	(No symbol) [0x005A82DC]
	(No symbol) [0x005A93DD]
	GetHandleVerifier [0x0094AABD+2539405]
	GetHandleVerifier [0x0098A78F+2800735]
	GetHandleVerifier [0x0098456C+2775612]
	GetHandleVerifier [0x007751E0+616112]
	(No symbol) [0x00685F8C]
	(No symbol) [0x00682328]
	(No symbol) [0x0068240B]
	(No symbol) [0x00674FF7]
	BaseThreadInitThunk [0x77337D59+25]
	RtlInitializeExceptionChain [0x77D2B74B+107]
	RtlClearBits [0x77D2B6CF+191]
">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: session not created
from no such execution context: uniqueContextId not found
  (Session info: chrome=114.0.5735.110)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-CF98PN2R', ip: '192.168.1.70', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_361'
Driver info: driver.version: Driver
remote stacktrace: Backtrace:
GetHandleVerifier [0x006EA813+48355]
(No symbol) [0x0067C4B1]
(No symbol) [0x00585358]
(No symbol) [0x005789B0]
(No symbol) [0x00577138]
(No symbol) [0x005777AA]
(No symbol) [0x00588417]
(No symbol) [0x005899E1]
(No symbol) [0x00589A80]
(No symbol) [0x005D1F1B]
(No symbol) [0x005D093C]
(No symbol) [0x005CA536]
(No symbol) [0x005A82DC]
(No symbol) [0x005A93DD]
GetHandleVerifier [0x0094AABD+2539405]
GetHandleVerifier [0x0098A78F+2800735]
GetHandleVerifier [0x0098456C+2775612]
GetHandleVerifier [0x007751E0+616112]
(No symbol) [0x00685F8C]
(No symbol) [0x00682328]
(No symbol) [0x0068240B]
(No symbol) [0x00674FF7]
BaseThreadInitThunk [0x77337D59+25]
RtlInitializeExceptionChain [0x77D2B74B+107]
RtlClearBits [0x77D2B6CF+191]

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:181)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:168)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)
at utils.Driver.getDriver(Driver.java:24)
at scripts.Base.setUp(Base.java:23)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:523)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase name="validatesubmission" time="4.102" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateContactUsInfo" time="4.285" classname="scripts._01_ProjectTest">
    <failure type="java.lang.AssertionError" message="expected [2860 S River Rd Suite 480, Des Plaines IL 60018] but found [2860 S River Rd Suite 480, Des Plaines, IL 60018]">
      <![CDATA[java.lang.AssertionError: expected [2860 S River Rd Suite 480, Des Plaines IL 60018] but found [2860 S River Rd Suite 480, Des Plaines, IL 60018]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at scripts._01_ProjectTest.validateContactUsInfo(_01_ProjectTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </failure>
  </testcase> <!-- validateContactUsInfo -->
  <system-out/>
  <testcase name="validatePhone" time="4.354" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateFullNameInputBox" time="4.350" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateConsentCheckBox" time="6.913" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateGenderRadioButton" time="13.621" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateEmail" time="0.271" classname="scripts._01_ProjectTest"/>
  <system-out/>
  <testcase name="validateTheFormSubmission" time="0.000" classname="scripts._01_ProjectTest">
    <skipped/>
  </testcase> <!-- validateTheFormSubmission -->
  <system-out/>
</testsuite> <!-- scripts._01_ProjectTest -->
